l10n/en-US/viewer.properties

```diff
@@ -168,20 +168,28 @@ find_match_case_label=Match case
 find_entire_word_label=Whole words
 find_reached_top=Reached top of document, continued from bottom
 find_reached_bottom=Reached end of document, continued from top
-# LOCALIZATION NOTE (find_matches_count): The supported plural forms are
+# LOCALIZATION NOTE (find_match_count): The supported plural forms are
 # [zero|one|two|few|many|other], with [other] as the default value.
 # "{{current}}" and "{{total}}" will be replaced by a number representing the
 # index of the currently active find result, respectively a number representing
 # the total number of matches in the document.
-find_matches_count={[ plural(n) ]}
-find_matches_count[one]={{current}} of {{total}} match
-find_matches_count[other]={{current}} of {{total}} matches
-# LOCALIZATION NOTE (find_matches_count_limit): The supported plural forms are
+find_match_count={[ plural(n) ]}
+find_match_count[zero]={{current}} of {{total}} matches
```

Comment:
This particular case should never happen, since when there's no matches the counter is hidden.

Reply:
Nope; please note that `total` must be non-zero for the message to be displayed in the first place, see https://github.com/mozilla/pdf.js/blob/master/web/pdf_find_bar.js#L162 and https://searchfox.org/mozilla-central/source/toolkit/content/widgets/findbar.js#1131.

I'll admit that this was perhaps not as clear as it could have been, given that the line in `pdf_find_bar.js` probably should have been e.g. `if (total > 0) {` instead.

Reply:
Done in the new commit.